frontend/components/Dashboard/Header/Header.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { black } from '../../../utils'
+import { navConstants } from '../Nav'
+
+interface Props {
+  onMenuClick: () => void
+}
+
+const Header: React.FC<Props> = ({ onMenuClick }) => {
+  return (
+    <div className="header">
+      <div className="hamburger-icon" onClick={onMenuClick}>
+        <div />
+        <div />
+        <div />
```

Comment:
haha these divs look funny 😆

Reply:
hahaha, prettier won't let me put them all in a line 😄 but i like that the DOM nodes actually visually look like a hamburger menu 😆